clients/src/main/java/org/apache/kafka/clients/tools/VerifiableProducer.java

```diff
@@ -124,6 +132,23 @@ private static ArgumentParser argParser() {
                 .choices(0, 1, -1)
                 .metavar("ACKS")
                 .help("Acks required on each produced message. See Kafka docs on request.required.acks for details.");
+        
+        parser.addArgument("--producer-prop")
```

Comment:
Other tools (at least producer performance, possibly others) just allow a list of `prop=value` arguments. Any reason not to be consistent with those (or whatever the KIP-4 discussion might have ended up agreeing upon)?